messages/tree.import.beta.md

```diff
@@ -0,0 +1,43 @@
+# summary
+
+Import data from one or more JSON files into an org.
+
+# description
+
+The JSON files that contain the data are in sObject tree format, which is a collection of nested, parent-child records with a single root record. Use the "<%= config.bin %> data export tree" command to generate these JSON files.
+
+If you used the --plan flag when exporting the data to generate a plan definition file, use the --plan flag to reference the file when you import. If you're not using a plan, use the --files flag to list the files. If you specify multiple JSON files that depend on each other in a parent-child relationship, be sure you list them in the correct order.
+
+# flags.files.summary
+
+Comma-separated and in-order JSON files that contain the records, in sObject tree format, that you want to insert.
```

Comment:
Haven't we been suggesting passing the flag multiple times with `arrayWithDeprecation` instead of separated values?